fixham/internal/operator/operator.go

```diff
@@ -0,0 +1,34 @@
+package operator
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/projectcalico/calico/fixham/internal/command"
+)
+
+const (
+	ImageName = "tigera/operator"
+)
+
+func GenVersion(rootDir, componentsVersionPath string) error {
```

Comment:
This is more of an optimization for the future, but `gen-versions` currently clones the `github.com/projectcalico/calico` repository as part of the generation. This is obviously a bit over-kill since we're running this from within the Calico repo already!

We should track a follow-on task to remove this sort of nested dependency.